test/e2e/adminUI/tests/group005Fields/text/uxTestTextField.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
 		browser
 			.end();
 	},
-	'Name field can be created via the initial modal': function (browser) {
+	'Text field can be created via the initial modal': function (browser) {
```

Comment:
As of yesterday, I started using the verb "filled" instead of "created" since I think it better matches what is doing.